api/src/main/java/org/openmrs/scheduler/db/SchedulerDAO.java

```diff
@@ -39,6 +39,17 @@
 	 */
 	public TaskDefinition getTask(Integer taskId) throws DAOException;
 	
+	/**
+	 * @since 2.3.0
```

Comment:
This is not a service layer interface. It is a data access layer interface, which is not directly accessed by clients.

Reply:
@dkayiwa  i have resolved the issue